component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateConstants.java

```diff
@@ -69,5 +69,7 @@ private X509CertificateConstants() {
             + "subjectRDN found for the given regex";
     public static final int MAX_ITEM_LIMIT_UNLIMITED = -1;
     public static final String SEARCH_ALL_USERSTORES = "SearchAllUserStores";
+    public static final String LOGIN_CLAIM_URIS = "LoginClaimURIs";
+    public static final String PRIMARY = "PRIMARY";
```

Comment:
No need to define a constant. Re-use [1]

[1] [1] https://github.com/wso2/carbon-kernel/blob/4.5.x/core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UserCoreConstants.java#L65